src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,53 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+
+    private String name = "Isosceles Trapezoid";
+    private String color;
+    private double firstLeg, secondLeg, thirdLeg, height, smallPiece;
+
+    public IsoscelesTrapezoid(double firstLeg, double secondLeg, double thirdLeg, String color) {
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+        this.thirdLeg = thirdLeg;
+        this.color = color;
+    }
+
+    @Override
+    public double findArea() {
+        if (firstLeg > secondLeg) {
+            smallPiece = (firstLeg - secondLeg) / 2;
+        } else {
+            smallPiece = (secondLeg - firstLeg) / 2;
+        }
```

Comment:
you don't need to get rid of minus, you use Math.pow ^2 anyway

Reply:
fix comment